_api-reference/profile.md

```diff
@@ -213,6 +221,220 @@ The response contains profiling information:
 ```
 </details>
 
+#### Example response: Concurrent segment search
+
+The following is an example response for a concurrent search with three segment slices:
```

Comment:
Confirm that "segment" shouldn't precede "search".

Reply:
Can be either way. The "segment" is understood in this case.

Reply:
Yes, but let's go for consistency 😄

Reply:
Sure, done!